app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java

```diff
@@ -46,8 +47,7 @@
 import static com.cornellappdev.android.eatery.model.enums.CampusArea.NORTH;
 import static com.cornellappdev.android.eatery.model.enums.CampusArea.WEST;
 
-public class MainActivity extends AppCompatActivity
-        implements MainListAdapter.ListAdapterOnClickHandler {
+public class MainActivity {
```

Comment:
Do we use this for anything??? Even so, I think we can merge this with main2activity and rename main2activity as main activity

Reply:
Oh ya this class isn't used I'll delete it

Reply:
let's rename main2activity to mainactivity!